### PR TITLE
Fix typos: No Source code changed, only comments, doc and test code

### PR DESCRIPTION
**Pull request by Sajjon**
Some typos fixed, no source **code** should have changed, only comments and documentation.

I did change name of some **test** code (misspelled function and variables names) thought.

There are a lot of typos in Localizable String files, alas, I refrained from changing them to get this PR merged first.

### CODE REVIEW

File: firefox-ios/firefox-ios-tests/Tests/XCUITests/FindInPageTests.swift
Comment (Sajjon) [inline firefox-ios/firefox-ios-tests/Tests/XCUITests/FindInPageTests.swift:1]: I've updated the test plan which referenced both renamed test methods

File: firefox-ios/firefox-ios-tests/Tests/ClientTests/ReaderModeStyleTests.swift [hunk 1]
```diff
@@ -81,7 +81,7 @@ class ReaderModeStyleTests: XCTestCase {
         XCTAssertNil(readerModeStyle)
     }
 
-    func test_initWithEmptyDictionnary_fails() {
+    func test_initWithEmptyDictionary_fails() {
```
Comment (Sajjon) [inline firefox-ios/firefox-ios-tests/Tests/ClientTests/ReaderModeStyleTests.swift:84]: Not referenced from any test plan

File: firefox-ios/firefox-ios-tests/Tests/ClientTests/ReaderModeStyleTests.swift [hunk 2]
```diff
@@ -72,7 +72,7 @@ class ReaderModeStyleTests: XCTestCase {
         XCTAssertEqual(readerModeStyle?.fontSize, ReaderModeFontSize.size1)
     }
 
-    func test_initWithWrongDictionnary_fails() {
```
Comment (Sajjon) [inline firefox-ios/firefox-ios-tests/Tests/ClientTests/ReaderModeStyleTests.swift:75]: Not referenced from any test plan

File: firefox-ios/firefox-ios-tests/Tests/ClientTests/ReaderModeStyleTests.swift [hunk 3]
```diff
@@ -61,7 +61,7 @@ class ReaderModeStyleTests: XCTestCase {
         )
     }
 
-    func test_initWithDictionnary_succeeds() {
+    func test_initWithDictionary_succeeds() {
```
Comment (Sajjon) [inline firefox-ios/firefox-ios-tests/Tests/ClientTests/ReaderModeStyleTests.swift:64]: Not referenced from any test plan

File: firefox-ios/firefox-ios-tests/Tests/ClientTests/OnboardingTests/OnboardingViewControllerProtocolTests.swift [hunk 4]
```diff
@@ -77,7 +77,7 @@ class OnboardingViewControllerProtocolTests: XCTestCase {
     }
 
     // MARK: - Test `getNextOnboardingCard` backwards
-    func testProtocol_getsNoViewContoller_beforeFirstCard() {
+    func testProtocol_getsNoViewController_beforeFirstCard() {
```
Comment (Sajjon) [inline firefox-ios/firefox-ios-tests/Tests/ClientTests/OnboardingTests/OnboardingViewControllerProtocolTests.swift:80]: Not referenced from any test plan

File: firefox-ios/firefox-ios-tests/Tests/ClientTests/OnboardingTests/OnboardingViewControllerProtocolTests.swift [hunk 5]
```diff
@@ -64,7 +64,7 @@ class OnboardingViewControllerProtocolTests: XCTestCase {
         XCTAssertEqual(result.viewModel.name, cards.sync.rawValue)
     }
 
-    func testProtocol_getsNoViewContoller_afterLastCard() {
+    func testProtocol_getsNoViewController_afterLastCard() {
```
Comment (Sajjon) [inline firefox-ios/firefox-ios-tests/Tests/ClientTests/OnboardingTests/OnboardingViewControllerProtocolTests.swift:67]: Not referenced from any test plan

File: firefox-ios/firefox-ios-tests/Tests/ClientTests/OnboardingTests/OnboardingButtonActionTests.swift [hunk 6]
```diff
@@ -67,7 +67,7 @@ class OnboardingButtonActionTests: XCTestCase {
         XCTAssertEqual(mockDelegate.action, OnboardingActions.readPrivacyPolicy)
     }
 
-    func testsubject_buttonAction_returnsNotifiactionsAction() {
+    func testsubject_buttonAction_returnsNotificationsAction() {
```
Comment (Sajjon) [inline firefox-ios/firefox-ios-tests/Tests/ClientTests/OnboardingTests/OnboardingButtonActionTests.swift:70]: Not referenced from any test plan

**Review by Sajjon — COMMENTED** [review]
Went through renamed test methods and updated the the only 2 referenced methods in a single test plan

**Comment by Sajjon** [comment]
@linabutler do you maintain this repo too? Any Action I need to take to get this merged?

**Review by linabutler — APPROVED** [review]
Looks great to me (I looked at every file! 😅), thank you so much for doing this, @Sajjon!

Would you mind rebasing this on the latest `main`, please? Then we can kick off CI, and merge if it’s green!

**Comment by Sajjon** [comment]
> Would you mind rebasing this on the latest `main`, please? Then we can kick off CI, and merge if it’s green!
@linabutler I think PR should be all good now

File: bitrise.yml [hunk 7]
```diff
@@ -314,7 +314,7 @@ workflows:
                         BUILD_SAMPLE_BROWSER_IOS="true"
                         ;;
                     BrowserKit*|taskcluster*|test-fixtures*)
-                        # Explicity directory changes - build all applications
+                        # Explicitly directory changes - build all applications
```
Comment (clarmso) [inline bitrise.yml:317]: 👍🏼

File: bitrise.yml [hunk 8]
```diff
@@ -1771,7 +1771,7 @@ workflows:
         title: ContentBlockerGen
 
   focus-pull-and-unzip-dependencies:
-    description: Pulls and unzip the dependencis from previous stage 
+    description: Pulls and unzip the dependencies from previous stage
```
Comment (clarmso) [inline bitrise.yml:1774]: 👍🏼

**Review by clarmso — APPROVED** [review]

**Review by adudenamedruby — APPROVED** [review]
Awesome. TYVM for your contribution!

**Comment by adudenamedruby** [comment]
tied to #20630

**Pull request merged**